source/utils/deprecation-warning.ts

```diff
@@ -0,0 +1,14 @@
+const alreadyWarned: string[] = [];
+
+export default (message: string) => {
+	if (alreadyWarned.includes(message)) {
+		return;
+	}
+
+	alreadyWarned.push(message);
+
+	// @ts-ignore Missing types.
```

Comment:
```suggestion
	// @ts-expect-error Missing types.
```

Reply:
Since this is as Typescript 3.9 feature and the repo is still using 3.8 I think it's better to leave it as it is.